modules/mono/glue/GodotSharp/GodotSharp/Core/AABB.cs

```diff
@@ -54,6 +54,15 @@ public Vector3 End
             set { _size = value - _position; }
         }
 
+        /// <summary>
+        /// The center of this <see cref="AABB"/>.
+        /// </summary>
+        /// <value>Equivalent to <see cref="GetCenter()"/>.</value>
+        public Vector3 Center
+        {
+            get { return GetCenter(); }
+        }
```

Comment:
Should add only one, a property or a method. Either choice is fine with me, but not both.

Reply:
Since,it's just a method in C++, I removed the property and kept only the method.


Btw, I did that to align with 
https://github.com/godotengine/godot/blob/2d1699ef82a29ef65d810194ee76dddc0d9e0389/modules/mono/glue/GodotSharp/GodotSharp/Core/Rect2.cs#L57-L65